modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java

```diff
@@ -68,9 +68,17 @@ protected void doProcessAction(
 		}
 
 		try {
+			int maxNumberOfPages = ParamUtil.getInteger(
+				actionRequest, "maxNumberOfPages");
+
+			if (maxNumberOfPages < 0) {
+				throw new PDFPreviewException(
+					"maximum-number-of-pages-limit-must-be-a-non-negative-" +
```

Comment:
@AliciaGarciaGarcia It is not necessary to translate the message or add it to the language files. A simple message suffices.

As this is not something that important, can you remove the last two commits and forward the pr directly? I'm ok with all other changes and tests were green.  Sorry for the misunderstanding.

Reply:
<img src="https://media4.giphy.com/media/67urFpVn7qwcd2gWIl/giphy.gif"/>